python/kvikio/kvikio/utils.py

```diff
@@ -131,8 +133,12 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def kvikio_deprecation_notice(msg: str) -> Callable:
-    """Decorate a function to print the deprecation notice at runtime.
+def kvikio_deprecation_notice(
+    msg: str = "This function is deprecated.",
+    deprecated_since_version: str = __version__,
```

Comment:
Make this a keyword-only argument and simplify the name. It will make the call sites more understandable.
```suggestion
    *, since: str = __version__,
```